app/helpers/smart_proxies_helper.rb

```diff
@@ -1,5 +1,5 @@
 module SmartProxiesHelper
-  TABBED_FEATURES = ["Puppet","Puppet CA"]
+  TABBED_FEATURES = ["Puppet", "Puppet CA", "Logs"]
```

Comment:
Yeah, that's caused by SQL server because this is what we do:

    proxy.features.where('features.name IN (?)', TABBED_FEATURES).uniq.pluck("name")

I can add a sort (by alphanum) and we can perhaps add a priority column to features that defines the ordering later?